### PR TITLE
Test suite for playlist mongo driver

### PR DESCRIPTION
**Pull request by Lacrymology**
Depends on #38

### CODE REVIEW

**Comment by Lacrymology** [comment]
This also fixes some implementation problems with the driver that became apparent when writing the tests

File: test/playlist-mongo-driver.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        self.db = mbc.db(conf.db);
+        self.driver.start();
+        self.from = moment(new Date());
+        self.to = moment((self.from.unix() + 120 * 60) * 1000); // add 2hs
+
+        var playlist = {
+            "_id": 1,
+	    "models" : [
+		{
+		    "file" : "/path/to/file1.avi",
```
Comment (jmrunge) [inline test/playlist-mongo-driver.js]: Please take a look at mvcp-driver-test to find out which files you can use (as they are created by "make test").

**Comment by jmrunge** [comment]
Really nice test suite!  Please fix the comment I told you and it will get merged! (After we're done with #38)

File: test/playlist-mongo-driver.js (hunk 1 above)
Comment (Lacrymology) [inline test/playlist-mongo-driver.js]: ok, I'll do that

But just by the way, these are never going to be played anyways

Comment (jmrunge) [inline test/playlist-mongo-driver.js]: I know, but just to be standard... Perhaps someday your test will be part
of another test... :)

2013/4/11 Tomas Neme notifications@github.com

**Comment by jmrunge** [comment]
@Lacrymology: Please next time be sure to add the required dependecies to package.json.  I had to edit it and add "should" to dev dependencies in order to be able to run "make test" (102 tests passed, by the way)

**Pull request merged**